server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java

```diff
@@ -65,6 +68,7 @@
 public class CacheManagerResource implements ResourceHandler {
```

Comment:
There are no tests or reproducers for the changes. Those tests should be in the ```rest``` module itself rather than outside.